ztp/source-crs/AddAdditionalNetworks.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
```

Comment:
Hi @karampok Thanks for this PR.
The Network CR is already included in the source-crs ([DisableSnoNetworkDiag.yaml](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/DisableSnoNetworkDiag.yaml#L2-L4)). Although the name is not generic you can add any content you want by using the overlay feature of the PolicyGenTemplate (PGT). 

Updating the reference group [PGT entry](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml#L97-L98) for DisableSnoNetworkDiag.yaml like this should be what you need:
```
    - fileName: DisableSnoNetworkDiag.yaml
      policyName: "config-policy"
      spec:
        additionalNetworks:
        - name: ran-mh-macvlan
          namespace: default
          rawCNIConfig: '{....}'
          type: Raw
```
This will add your additionalNetworks entry to the default content from the source-cr (you can also overwrite the `disableNetworkDiagnostics` entry if desired/necessary).
Hope this helps!